frontend/src/static/js/components/webstatus-overview-page.ts

```diff
@@ -89,6 +105,54 @@ export class OverviewPage extends LitElement {
         }
       },
     });
+
+    // Set up listener of 'exportToCSV' event from webstatus-overview-filters.
+    this.addEventListener('exportToCSV', () => {
+      this.exportToCSV();
+    });
+  }
+
+  async exportToCSV(): Promise<void> {
+    if (!this.allFeaturesFetcher) {
+      return;
+    }
+    // Fetch all pages of data via getAllFeatures
+    const allFeatures = await this.allFeaturesFetcher();
+    const columns = [
```

Comment:
One thing to note. You should use the CELLS_DEF for information about the columns:

https://github.com/GoogleChrome/webstatus.dev/blob/944256342539ff716b25b845c4a0503fe7e5a511/frontend/src/static/js/components/webstatus-overview-cells.ts#L194-L258


Additionally, to get the current columns, use `parseColumnsSpec`. That way you can get the active columns